examples/tfengine/generated/multi_envs/audit/terraform.tfvars

```diff
@@ -15,3 +15,17 @@
 
 folder         = "folders/12345678"
 auditors_group = "example-auditors@example.com"
+additional_filters = [
+  "logName=\"logs/forseti\"",
```

Comment:
let's change the = signs to : to be consistent with the default one, i.e. logName:\"logs/cloudaudit.googleapis.com\"

Reply:
Done. I also did `terraform apply` with the new change and everything seems to be fine